cypress.config.ts

```diff
@@ -22,6 +22,7 @@ const baseHost = getEnvVar('E2E_BASE_HOST', 'SPRYKER_FE_HOST');
 export default defineConfig({
   env: {
     repositoryId: process.env.ENV_REPOSITORY_ID,
+    isDynamicStoreEnabled: getEnvVar('ENV_IS_DYNAMIC_STORE_ENABLED', 'SPRYKER_DYNAMIC_STORE_MODE') || false,
```

Comment:
Please check how it was done in DMS PR

https://github.com/spryker/cypress-tests/pull/17/files#diff-3f62e885350a48939f240b9daf40f6b06d41da7751f2b613238fc7669ac0a1b2R25

Reply:
fixed